index.html

```diff
@@ -18,26 +21,170 @@ <h2 class="log"><a class="Welcome" href="#">Welcome</a></h2>
         </nav>
     </header>
     <main>
-    <section class="section-1"> 
-        <h1 class="heading-1">Hey there. I’m Josphat</h1>
+        <section class="section-1">
+            <h1 class="heading-1">Hey there. I’m Josphat</h1>
             <h2 class="heading-2">I’m a software developer</h2>
-                <p class="par-1"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-            don’t hestiate to
-                </p>
-        <ul class="icons">
-            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
-            <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
-            <li><i class="fa fa-medium" aria-hidden="true"></i></li>
-            <li><i class="fa fa-github" aria-hidden="true"></i></li>
-            <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
-        </ul>
-    </section>
-</main>
+            <p class="par-1"> I can help you build a product , feature or website Look through some of my work and
+                experience! If you like what you see and have a project you need coded,
+                don’t hestiate to
+            </p>
+            <ul class="icons">
+                <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+                <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+                <li><i class="fa fa-medium" aria-hidden="true"></i></li>
+                <li><i class="fa fa-github" aria-hidden="true"></i></li>
+                <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
+            </ul>
+            <div class="down-img">
+                <img src="img/Disabled.png" class="disable" alt="disble">
+            </div>
+        </section>
+
+        <section class="work-section">
+            <div class="section-title">
+                <h2>My recent Works</h2>
+            </div>
+            <div class="green-line">
+                <img src="img/Indicator.png" alt="indicator">
```

Comment:
Kindly explain what you mean by this because I see nothing wrong with how it was implimated. Please shade more light. i'll really appreciate the feedback.

Reply:
No problem @zairdon20 ,  I'll ask you to please do a quick read on this article: https://www.w3docs.com/snippets/html/when-to-use-html-img-tag-and-css-background-image-property.html , let me know if you still have any questions about why you should have this as a bg instead of an `<img>`, it mostly boils down to performance, and download time, as well as being best practices for decorations as it enables transitions and animations easier, I do understand that this might not look relevant for this project, but a lot of requirements like "Use grid on this section" or "Align this with flex" are asked from you simply because of the learning value, the use of semantic HTML is the same thing, it might not look relevant or that there is nothing wrong with it right now, but knowing what best practices are in a learning environment, even if in a particular personal or commercial project in the future you decide that it's not the best approach is very important and it's one of the goals of the HTML module.

Let me know if you have any other questions

Best, 
Leo